app/src/main/java/com/android/shelfLife/ui/overview/Overview.kt

```diff
@@ -37,8 +37,10 @@ import kotlinx.coroutines.launch
  * @param navigationActions The actions to handle navigation
  */
 @Composable
-fun OverviewScreen(navigationActions: NavigationActions) {
-  val overviewScreenViewModel = hiltViewModel<OverviewScreenViewModel>()
+fun OverviewScreen(
```

Comment:
## Nice work refactoring
- As discussed in our stand up meetings, we agreed to have the hiltViewModels be (default initialised) parameters to the Screens, so that in the testing we can pass some test viewModels; good job applying this